main_f1.c

```diff
@@ -128,7 +124,7 @@ board_deinit(void)
 static inline void
 clock_init(void)
 {
-#if defined(INTERFACE_USB)
+#if INTERFACE_USB
 	rcc_clock_setup_in_hsi_out_48mhz();
 #else
 	rcc_clock_setup_in_hsi_out_24mhz();
```

Comment:
I am not sure if I need to change the clock setup for the MAV Station. As far as I know, it should also be able to use the USART BL if the clock is setup for the USB.